examples_utils/src/camera.rs

```diff
@@ -0,0 +1,141 @@
+use bevy::input::mouse::MouseMotion;
+use bevy::prelude::*;
+
+pub struct CameraControllerPlugin;
```

Comment:
This needs some documentation. It doesn't have to be much, but just explaining that this is a simple fly camera.

The scene_viewer examples prints the camera controls, but I think this plugin should be in charge of that. Otherwise it will be easy to forget to update it.